Automattic/jetpack
# Subscriptions: add a Subscribers menu to the dashboard

## Proposed changes:

The menu will only appear on self-hosted sites, since WordPress.com sites (simple and WoA) already get a Subscribers menu under "Users".

~That menu will either link to Calypso (by default), or to the Stats screen when the Stats module is enabled and Odyssey stats are enabled.~
That menu will always link to Calypso, and will appear if your site is connected to WordPress.com, not offline, and when the Subscriptions module is active.

### Other information:


## Jetpack product discussion

* p1688642967595329/1688481163.954019-slack-C0299DMPG
* pbNhbs-7bW-p2
* pbNhbs-7F3-p2#comment-17295

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

Test this on a self-hosted site, connected to WordPress.com, with the Subscriptions module active.

* You should see a Subscribers menu under Jetpack.
* ~That menu should lead you to a different place whether the Stats module is active or not.~
* ~When linking to wp-admin, it should open the Subscribers tab of the Stats screen.~
* That menu should lead to the relevant Subscribers page in Calypso
